fastlane/lanes/release.rb

```diff
@@ -148,21 +163,30 @@
     android_finalize_prechecks(skip_confirm: options[:skip_confirm])
     configure_apply(force: is_ci)
 
+    app_version = android_get_app_version()
+    release_branch = "release/#{app_version}"
+
+    # Remove branch protection first, so that we can push the final commits directly to the release branch
+    removebranchprotection(repository: GHHELPER_REPO, branch: release_branch)
+
     check_translations_coverage()
     download_translations()
 
     android_bump_version_final_release()
     version = android_get_release_version()
     download_metadata_strings(version: version['name'], build_number: version['code'])
 
+    push_to_git_remote(tags: false)
+
     # Wrap up
-    removebranchprotection(repository: GHHELPER_REPO, branch: "release/#{version['name']}")
     setfrozentag(repository: GHHELPER_REPO, milestone: version['name'], freeze: false)
     create_new_milestone(repository: GHHELPER_REPO)
     close_milestone(repository: GHHELPER_REPO, milestone: version['name'])
 
     # Trigger release build
     trigger_release_build(branch_to_build: "release/#{version['name']}")
+
+    create_release_management_pull_request('trunk', "Merge #{app_version} final to trunk")
```

Comment:
Same rationale [as above](https://github.com/wordpress-mobile/WordPress-Android/pull/18398/files#r1191844874).

```suggestion
    create_release_management_pull_request('trunk', "Merge #{app_version} final")
```

Reply:
Same rationale [as above](https://github.com/wordpress-mobile/WordPress-Android/pull/18398#discussion_r1191862103). 😅